cypress/component/DarCollectionTable/actions.spec.js

```diff
@@ -0,0 +1,212 @@
+/* eslint-disable no-undef */
+import { React } from 'react';
+import { mount } from 'cypress/react';
+import Actions from '../../../src/components/dar_collection_table/Actions';
+import {cloneDeep} from 'lodash/fp';
+import { Navigation } from '../../../src/libs/utils';
+import { Storage } from '../../../src/libs/storage';
+
+let propCopy;
+const collectionId = 1;
+
+const darColl = {
+  'darCollectionId': collectionId,
+  "referenceIds": [
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9883624)
```suggestion
  'referenceIds': [
```